config/locales/en.yml

```diff
@@ -1626,11 +1626,14 @@ en:
       befriend_them: "You can also add them as a friend at %{befriendurl}."
       befriend_them_html: "You can also add them as a friend at %{befriendurl}."
     gpx_description:
+      description_with_tags: "It looks like your GPX file %{trace_name} with the description %{trace_description} and the following tags: %{tags}"
       description_with_tags_html: "It looks like your GPX file %{trace_name} with the description %{trace_description} and the following tags: %{tags}"
```

Comment:
In the html version the most important link is the link to the trace page (`/user/:user/traces/:id`), which is inserted at `%{trace_name}`. Plaintext version doesn't have this link, `%{trace_name}` is just the filename. Maybe we can rewrite `description_with_tags` and `description_with_no_tags` to include the url? Then they won't be just copies without the `_html` suffixes.

Reply:
Yes, good point, I have added `trace_url` now. The html version somehow assumes this URL could be null, which is not clear to me.

Reply:
This can happen on failure when `@trace_url` is not set. You'll get a plaintext message like "... It looks like your GPX file Trace 1.gpx at  with the description ...". I guess we need two plaintext locale strings, with an url and without it.

Reply:
I added some new text "The trace is available for download at %{trace_url}" that is only shown in case of success, and removed the %{trace_url} again from description_with(_no)?_tags. I think this is a bit cleaner, and also easier to read as an email.

Reply:
The logic of this email, I assume, is not to tell the user how to download a file, but to tell which file was processed. The filename is likely generated by whatever track writing software and may not be easily recognizable, that's why we add an url. The user doesn't need to download the file because they've just uploaded it.

But if you think that your changes are easier to read, shouldn't the html version be also modified in this manner?

Reply:
The download URL that is shown in the email is controlled by the Rails port, not the user uploading the trace. Here's an example:

```
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

Hi mmd2,

  It looks like your GPX file Track_2014_06_13_184543.gpx with the description dsfasf and the following tags: d

loaded successfully with 3427 out of a possible 3427 points.

The trace is available for download at http://openstreetmap.example.com/user/mmd2/traces/64

All your successfully uploaded GPX traces can be found at http://openstreetmap.example.com/traces/mine

```

Reply:
Updated text email:

```
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

Hi mmd2,

  It looks like your GPX file Track_2014_06_13_184543.gpx with the description dsfsdf and the following tags: sdfsdf

loaded successfully with 3427 out of a possible 3427 points.

Your trace is available at http://openstreetmap.example.com/user/mmd2/traces/65

All your successfully uploaded GPX traces can be found at http://openstreetmap.example.com/traces/mine

```